tests/helper_methods.py

```diff
@@ -0,0 +1,21 @@
+import importlib
+
+import pytest
+
+
+def skip_if_missing(name: str) -> pytest.mark.skip:
+    """Ignore tests which do not have the package installed.
+
+    Notes
+    -----
+    This might be usable as a pytest plugin in the future, if someone builds
```

Comment:
See this note please. I think this is the simplest implementation of checking if a package is missing.